crypto/engine/eng_devcrypto.c

```diff
@@ -1156,6 +1158,9 @@ void engine_load_devcrypto_int()
     ENGINE *e = NULL;
 
     if ((cfd = open("/dev/crypto", O_RDWR, 0)) < 0) {
+#ifndef ENGINE_DEVCRYPTO_DEBUG
+        if (errno != ENOENT)
+#endif
         fprintf(stderr, "Could not open /dev/crypto: %s\n", strerror(errno));
```

Comment:
IMHO the `fprintf` should be indented because the if statement is present by default. So the current indentation is misleading.

Reply:
Done.